### ISSUE TITLE
Press Coverage

### ISSUE THREAD

**Issue by hugovk** [dariusk/NaNoGenMo-2015#9]
A thread to collect any notable press coverage this year. 

Here's last year's: https://github.com/dariusk/NaNoGenMo-2014/issues/92

**Comment by enkiv2** [comment]
I guess blog posts count as press coverage? I wrote this a while back: https://medium.com/@enkiv2/the-hidden-benefits-of-nanogenmo-dd91193bda6#.d3cy9qbwr

**Comment by whitten** [comment]
Thank you for the Interesting blog article. You referenced something called a "cut-up" what is it?

**Comment by ikarth** [comment]
Slightly off-topic for this thread, but the [cut-up technique](https://en.wikipedia.org/wiki/Cut-up_technique) is a Dadaist method of generating text. There's a number of past artistic movements--including Dada, Surrealism, and OuLiPo--that are relevant to text generation.

On topic, if blog posts count, I've been [covering past entries on my procedural generation blog for a while](http://procedural-generation.tumblr.com/tagged/nanogenmo).

**Comment by enkiv2** [comment]
Information on cut-up techniques might actually be on-topic for the
resources thread. While they've been around for a while (even in the
context of generative text), a lot of people are coming at this without a
strong background in the history of experimental and semi-mechanical text
generation (along with the movements that produced some of the forms of
constrained fiction that a lot of generative text experiments come out of).

**Comment by dariusk** [comment]
We got a mention in the Fast Forward Labs machine learning newsletter:
http://us8.campaign-archive2.com/?u=bdb368b9a389b010c19dbcd54&id=e924d57988

**Comment by samplereality** [comment]
Not exactly press coverage, but we'll be talking about NaNoGenMo in [my online class about electronic literature](https://www.edx.org/course/electronic-literature-davidsonx-d004x) this coming week. Guest appearance on Thursday by @dariusk via a Google Hangout!

**Comment by cpressey** [comment]
Does [this](http://paper.li/ActiveNick/1330052730?edition_id=a066f8a0-8a39-11e5-852b-0cc47a0d15fd) count?  (Scroll down, or click the "Business" tab.)

**Comment by hugovk** [comment]
Nope, it's a (spammy) auto-generated thing that just points to this repo.

**Comment by ikarth** [comment]
> Nope, it's a (spammy) auto-generated thing that just points to this repo.

New idea: a generator that writes news coverage about NaNoGenMo. There should be enough from the past couple of years to make a nice little source corpus.

**Comment by enkiv2** [comment]
Oh god. I will admit, most press coverage on generative text is pretty
formulaic. Bonus points if we take advantage of a large database of
entirely out of context quotes from Darius and Nick Montfort.

**Comment by MichaelPaulukonis** [comment]
@hugovk  auto-generated texts are _the worst_!

**Comment by tra38** [comment]
> Nope, it's a (spammy) auto-generated thing that just points to this repo.

Paper.li uses automation to engage in content curation, which is essential because there's too much content on the Internet as it is. It provides a valuable service, and is not particularly spammy (even though there are a lot of 'inactive' Paper.li newsletters that just keep on being generated with no human readers, and the periodic tweets that it sends out to its unwitting "human" curators can be incredibly annoying).

**Comment by hugovk** [comment]
https://paper.li/stop-mentions.html

**Comment by enkiv2** [comment]
Here's some actually machine-generated low-quality clickbait articles about
NaNoGenMo: https://github.com/enkiv2/NaNoGenMo-2015/blob/master/clickbait.md

**Comment by dariusk** [comment]
Oh wow I hate paper.li! Discovering their "stop mentions" url a few years
ago (which @Hugovk links above) was very important.

**Comment by hugovk** [comment]
# NaNoGenMo: Dada 2.0

http://arcade.stanford.edu/blogs/nanogenmo-dada-20

Cites stuff by @dariusk, @enkiv2, @aparrish, @ikarth, @thricedotted, @michelleful, @puckey.

**Comment by hugovk** [comment]
# The Complexity of Machine Writing

http://digitalfellows.commons.gc.cuny.edu/2015/11/30/nanogenmo/

by @jeffbinder

**Comment by hugovk** [comment]
# The art of the Twitter bot

http://www.alphr.com/twitter/1002130/the-art-of-the-twitter-bot/page/0/1

Interview with @v21 which also covers NaNoGenMo and cites https://github.com/dariusk/NaNoGenMo-2015/issues/165 by @leonardr.

**Comment by hugovk** [comment]
# NaNoGenMo CORNER!

http://www.rockpapershotgun.com/2015/12/05/best-free-pc-games-of-the-week-7/

Cites stuff by @lorenSchmidt and @kevandotorg.

**Comment by hugovk** [comment]
# Computers Get Busy for National Novel-Generating Month

http://thenewstack.io/computers-get-busy-national-novel-generating-month/

Cites https://github.com/dariusk/NaNoGenMo-2015/issues/190 by @adregan, and stuff by @lizadaly, @MichaelPaulukonis, amongst others.

**Comment by hugovk** [comment]
# NaNoGenMo : pourra-t-on générer automatiquement le prochain prix Goncourt ?

http://www.numerama.com/pop-culture/135762-peut-on-generer-automatiquement-le-prochain-chef-doeuvre-de-la-litterature.html

In French, cites #190 by @adregan, https://github.com/dariusk/NaNoGenMo-2015/issues/160 by @VincentToups and https://github.com/dariusk/NaNoGenMo-2015/issues/186 by me.

**Comment by MichaelPaulukonis** [comment]
# 500 computer-generated novels: the Nanogenmo 2015 entrants

http://boingboing.net/2015/12/21/500-computer-generated-novels.html

NB: I am a frequent commenter in the BB forums, although the poster, Cory Doctorow, does not pay much attention to the forums.

**Comment by hugovk** [comment]
I've no idea where that 500 came from, perhaps it's a typo due to the 50k-words requirement. 
Anyway, it's via Slashdot:

# Programmers Share 188 Computer-Generated Novels On GitHub

http://developers.slashdot.org/story/15/12/20/2228211/programmers-share-188-computer-generated-novels-on-github

---

(It's hard to know how many novels were generated. There are [188 open](https://github.com/dariusk/NaNoGenMo-2015/issues) and [four closed issues](https://github.com/dariusk/NaNoGenMo-2015/issues?q=is%3Aissue+is%3Aclosed) in this repo. Of the 188, [six are admin issues](https://github.com/dariusk/NaNoGenMo-2015/issues?q=is%3Aopen+is%3Aissue+label%3Acompleted+label%3Aadmin) and [79 are labelled complete](https://github.com/dariusk/NaNoGenMo-2015/issues?q=is%3Aopen+is%3Aissue+label%3Acompleted). Some issues are unfinished declarations of intent, some have single completed projects, some have multiple. Some completed ones generated more than one novel, and of course more were generated and discarded during development. We also know [152 people declared intent](https://twitter.com/NaNoGenMoBot/status/675764976884310016): this is the count of unique authors of non-admin issues.)

**Comment by hugovk** [comment]
# “Brass. Brass. Brass.”: Beim NaNoGenMo werden Algorithmen zum Roman-Autor

http://www.e-book-news.de/nanogenmo-literaturproduktion-per-algorithmus/

In German, cites #190 by @adregan, https://github.com/dariusk/NaNoGenMo-2015/issues/184 by @spc476 and https://github.com/dariusk/NaNoGenMo-2015/issues/142 by @kevandotorg.

**Comment by adregan** [comment]
Does all this press coverage in late December mean that all of these journalist finally finished reading all the novels?

**Comment by ikarth** [comment]
> I've no idea where that 500 came from, perhaps it's a typo due to the 50k-words requirement. 

I mean, technically we could output infinite novels if we really wanted to.

**Comment by tra38** [comment]
# Algo-Poetry Generation Month

http://www.nerdcore.de/2015/12/23/algo-poetry-generation-month/#.VoVV_oQc-AY

In German. Adds #180 and #109 to the list of cited works, among other previously-cited literature.

**Comment by tra38** [comment]
# Another Word: Let's Write a Story Together, MacBook

http://clarkesworldmagazine.com/another_word_01_16/

First official press coverage of #11 .

**Comment by mewo2** [comment]
The bot version of #156 has been getting some press coverage: [Boing Boing](http://boingboing.net/2016/08/11/generate-your-own-random-fanta.html), [Wired](http://www.wired.com/2016/08/bot-tweets-totally-fantastical-map-every-hour/), [National Geographic](http://news.nationalgeographic.com/2016/08/bot-fantasy-map-generator/). I even managed to talk a bit about NaNoGenMo in the National Geographic interview!

**Comment by ikarth** [comment]
It deserves the attention. Especially your write-ups on how it works.
